webui/src/lib/api/index.js

```diff
@@ -594,20 +595,20 @@ class Tags {
 
 class Objects {
 
-    async list(repoId, ref, tree, after = "", amount = DEFAULT_LISTING_AMOUNT, delimiter = "/") {
-        const query = qs({prefix: tree, amount, after, delimiter});
+    async list(repoId, ref, tree, after = "", presign= false, amount = DEFAULT_LISTING_AMOUNT, delimiter = "/") {
```

Comment:
For the next time:
- Prefer to add new defaults at the end - as it not require going all over the callers and make sure we fix the call arg order.
- This call already gets a lot of arguments (usually my number is 7) where we can capture a lot of the arguments into options object. In this case all the defaults can be set by 'options' objects. The caller call will be immune to arg order bugs and will make the code more readable.

Reply:
I added the new default not at the end because I wanted to pass a parameter to it and if it is in the end you need to pass a parameter to all other defaults before, didn't find another way to do it in js

Reply:
so using options will be the way